From 144c76bb3f2cf7c844db4d73030a1f1447322df0 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Fri, 8 Dec 2017 11:44:47 +0800
Subject: [PATCH 3013/5242] MLK-16976-5 ARM64: dts: fsl-imx8qxp-mek: add 12V
 source capability for Type-C

commit  a863a741b44c12fff00e8f63ef6ccea439f3b2b8 from
https://source.codeaurora.org/external/imx/linux-imx.git

The MEK hardware design supports 12V power source, update src-pdos for it.
And It doesn't support power sink, so remove additional sink PDOS setting.

BuildInfo:
- SCFW 245582b, IMX-MKIMAGE 0ad6069a, ATF 6bd98a3
- U-Boot 2017.03-imx_v2017.03+gfa65b0a

Acked-by: Jun Li <jun.li@nxp.com>
Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index 8352967..400536b 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -652,8 +652,8 @@
 		interrupts = <3 IRQ_TYPE_LEVEL_LOW>;
 		ss-sel-gpios = <&gpio5 9 GPIO_ACTIVE_LOW>;
 		reset-gpios = <&pca9557_a 7 GPIO_ACTIVE_HIGH>;
-		src-pdos = <0x380190c8>;
-		snk-pdos = <0x380190c8 0x3802d0c8>;
+		src-pdos = <0x380190c8 0x3803c0c8>;
+		snk-pdos = <0x380190c8>;
 		max-snk-mv = <9000>;
 		max-snk-ma = <1000>;
 		op-snk-mw = <9000>;
-- 
1.7.9.5

